wpuf-functions.php

```diff
@@ -3880,3 +3880,30 @@ function wpuf_user_has_roles( $roles, $user_id = 0 ) {
 
     return false;
 }
+
+/**
+ * Create a private page for form preview
+ *
+ * @return bool|false|string|WP_Error
+ */
+function get_wpuf_preview_page(){
```

Comment:
@zamandevs spacing issues detected in this function. Please add correct spacing.